### PR TITLE
DISTMONGO-158 Reworked installation doc

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   requirements.txt
	modified:   source/conf.py
	modified:   source/installation.rst
	new file:   source/repo-overview.rst
modified:   source/index.rst
	modified:   source/repo-overview.rst

### CODE REVIEW

File: source/installation.rst [hunk 1]
```diff
@@ -1,116 +1,128 @@
 .. _install:
 
 =============================================     
-Installing |pdmdb|
+Install |pdmdb|
 =============================================
 
-.. contents::
-   :local:
+We recommend to install |pdmdb| from Percona repositories using the package manager of your operation system. Find the list of supported Linux distributions on the `Percona Software and Platform Lifecycle <https://www.percona.com/services/policies/percona-software-platform-lifecycle#mongodb>`_ page.
```
Comment (spron-in) [inline source/installation.rst]:
```suggestion
We recommend to install |pdmdb| from Percona repositories using the package manager of your operating system. Find the list of supported Linux distributions on the `Percona Software and Platform Lifecycle <https://www.percona.com/services/policies/percona-software-platform-lifecycle#mongodb>`_ page.
```

File: source/installation.rst [hunk 2]
```diff
@@ -1,116 +1,128 @@
...
+         
+         Percona provides :ref:`two repositories <repo-overview>` for |pdmdb|. To enable a repo, we recommend using the ``setup`` command:
 
-Install on *Debian / Ubuntu*
-----------------------------------------------
+         .. code-block:: bash
 
-Use the following commands to install |PSMDB| and / or
-|PBM|:
+            $ sudo percona-release setup pdmdb-4.4
```
Comment (spron-in) [inline source/installation.rst:80]: Do we have a page somewhere describing what other versions do we support and how to set it up?

Like pdmdb-5.0 I would assume is the Distribution for 5.0.

File: source/installation.rst (hunk 1 above)
Comment (nastena1606) [reply source/installation.rst]: Updated

File: source/installation.rst (hunk 2 above)
Comment (nastena1606) [reply source/installation.rst:80]: These are in the corresponding version's docs.

**Pull request merged**